### ISSUE TITLE
Add Campaign URL to InvalidPhotoMessage

### ISSUE THREAD

**Issue by jamjensen** [DoSomething/gambit#828]
_FEATURE OVERVIEW_
-------------------
#### User Story
- As a user I'd like to be able to click the campaign url if I'm unable to send a photo over MMS so that I can still reportback on the campaign. 

#### Why This Matters
- There are ~70 users who have completed Give A Spit (from the broadcast on March 21) that are unable to send us their photo. They try to send their photo and receive the InvalidPhotoMessage. The current InvalidPhotoMessage gives them 2 options: (1) try again to text back their photo; (2) text back Q to ask a question. We then see users texting in Q and asking us how to reportback. Currently, the best way to help them is for an admin to send them a link to the user's campaign. 

#### Acceptance Criteria
- **Given** I receive the InvalidPhotoMessage
- **When** I click the URL to the campaign page
- **Then** I am able to upload a RB for the campaign I interacted with over SMS

**Comment by aaronschachter** [comment]
This can be overridden per Campaign @jamjensen so for Give A Spit, we can override the No Photo Sent message with a URL. Is this issue to implement a new Liquid style variable tag like `{{campaign_url}}`?

**Comment by jamjensen** [comment]
@aaronschachter that sounds like the best way to accomplish the goal, so yes -- this issue requests that we implement a new Liquid style variable tag. 

If we were to implement a new liquid style tag would that be something we can use when we're off of Mobile Commons?

**Comment by aaronschachter** [comment]
Yup, Gambit is coded to use the Liquid style `{{variable_name}}` just for the sake of familiarity -- it doesn't really ever touch Mobile Commons Liquid.

It sounds like what would really be the jam for the Campaign URL variable is a [magic link](https://github.com/DoSomething/northstar/blob/7542a6688675b67fba3290a2a42254c3c68fce3f/documentation/endpoints/legacy/auth.md#create-phoenix-session) like we use on the Mobile App, so when the user clicks on the link they're already signed in. We'll want to check with the Northstar team though, as the mobile app uses a legacy Northstar endpoint to accomplish this.

**Comment by aaronschachter** [comment]
Labelled this with blocked, waiting on switch to Phoenix from Ashes (#1019)
